### ISSUE TITLE
Disambiguate between "two X" and "alternative versions of X"

### ISSUE THREAD

**Issue by tychonievich** [FamilySearch/GEDCOM#269]
7.0 does not have a way to assert "only one of these is right, we don't know which one." That is a feature that would likely be useful, e.g. for conflicting records of events, attributes, or even individuals or citation data.

_Originally posted by @tychonievich in https://github.com/FamilySearch/GEDCOM/issues/253#issuecomment-1402734563_

**Comment by Norwegian-Sardines** [comment]
I agree that, at the very least, an assertion of a fact should have some type of qualifier when a duplicate fact is also present in the GEDCOM.

Source_Citation has a `QUAY` tag that indicates the credibility of a source associated with an asserted fact but this indication does not place any qualifiers on the actual fact!  For instance: two normally and equally reliable sources could assert different dates, places or any other element of a fact during the process of researching and collecting information, producing two facts of the same type. Or two similar facts/events or people could be asserted but actually be two different instance.

Some indicator with notes should be developed to a) indicate that only one can be correct but can not be established until more evidence is collected, b) both are correct indicating different events or attributes, c) this specific one is correct, others are not, d) this specific one is not correct, another one is correct.

option (d) is valuable as a way to “show our work” that the assertion is incorrect (with notes) and stem a potential discussion down the road by others who believe in “alternate facts”!
